setup.py

```diff
@@ -0,0 +1,23 @@
+import os
+from setuptools import setup
+
+PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+
+def _load_readme():
+    readme = os.path.join(PROJECT_ROOT, "README.md")
+    with open(readme) as f:
+        return f.read()
+
+
+setup(
+    name="configsuite-tui",
+    author="Software Innovation Bergen, Equinor ASA",

```

Comment:
Well, since your not part of Software Innovation Bergen, I suggest we leave this as Equinor ASA 🤷